chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -35,6 +35,7 @@ def self.get_network_by_type(node, type)
               return Network.new(net, data, intf, interface_list)
             end
           end
+          return nil
```

Comment:
Yes, for this
https://github.com/crowbar/barclamp-provisioner/pull/459/files#diff-531ca46b175571ed356cbc6057f38685R139